cluster-autoscaler/README.md

```diff
@@ -122,8 +122,8 @@ default deployment strategy on GCP.
 It is possible to run a customized deployment of Cluster Autoscaler on worker nodes, but extra care needs
 to be taken to ensure that Cluster Autoscaler remains up and running. Users can put it into kube-system
 namespace (Cluster Autoscaler doesn't scale down node with non-mirrored kube-system pods running
-on them) and add `scheduler.alpha.kubernetes.io/critical-pod` annotation (so that the rescheduler,
-if enabled, will kill other pods to make space for it to run).
+on them) and set a `priorityClassName: system-cluster-critical` property on your pod spec
+(so that the rescheduler, if enabled, will kill other pods to make space for it to run).
```

Comment:
Can we remove the part about rescheduler as well? Rescheduler was the thing using the critical-pod annotation that has been deperecated, it has nothing to do with priority classes.

Reply:
Good catch thanks!
I changed it to `to prevent your pod from being evicted`.